code/src/components/QuestionAboutAuthor.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+
+const QuestionAboutAuthor = ({ onFormSubmit }) => {
+  const [select, setSelect] = useState("");
+  const [alert, setAlert] = useState(false);
+  const answerList = [
+    "Jacqueline",
+    "Jane",
+    "Jennifer",
+    "Jody",
+    "Julie",
+    "Joanne",
+  ];
+
+  const onAlert = () => {
+    setAlert(true);
+    setTimeout(() => {
+      setAlert(false);
+    }, 2000);
+  };
```

Comment:
How is this alert being used? I tried to trigger it on my browser but couldn't get it to work